example/1_gemm_xdl/gemm_xdl_int8.cpp

```diff
@@ -25,12 +25,14 @@ using F32 = float;
 using Row = ck::tensor_layout::gemm::RowMajor;
```

Comment:
Quantized GEMM should not be considered as "GEMM", it's GEMM fusion.

So please rename this to cpp and put it in a different example folder in next PR